api/src/lib/aws.ts

```diff
@@ -97,13 +97,21 @@ async function sendHeadObjectToS3Bucket(bucketName: string, key: string) {
   await s3.send(new HeadObjectCommand(uploadParams))
 }
 
+export function getS3UploadFileKey(organizationId: number, upload: Upload | CreateUploadInput, uploadId?: number) {
+  if ('id' in upload) {
```

Comment:
Could we add a small explanatory comment here for when we'd expect this to hit vs the `uploadId` we pass in?